### PR TITLE
Support local files

### PR DESCRIPTION
**Pull request by joshmoore**
Unfortunately, `requests` doesn't (easily) handle local paths or the
file:// scheme. Instead, we perform a check and delegate to an
underlying implementation.

### CODE REVIEW

**Comment by will-moore** [comment]
This should fix travis https://github.com/will-moore/ome-zarr-py/commit/d7469f926d651b1515ea10314225b5085b3318f9

File: ome_zarr.py [hunk 1]
```diff
@@ -44,71 +49,112 @@ def napari_get_reader(path: PathLike) -> Optional[ReaderFunction]:
...
+
+    def reader_function(self, path: PathLike) -> List[LayerData]:
+        """Take a path or list of paths and return a list of LayerData tuples."""
+        if isinstance(path, list):
+            path = path[0]
+        return [self.load_omero_zarr()]
+
+    def load_omero_metadata(self):
+        """Load OMERO metadata as json and convert for napari"""
+        metadata = {}
```
Comment (will-moore) [inline ome_zarr.py:93]:
```suggestion
        metadata = {}
        image_data = self.image_data
```

Comment (will-moore) [reply ome_zarr.py:93]: This will fix travis

Comment (joshmoore) [reply ome_zarr.py:93]: Went ahead and inlined `channels` as well.

**Comment by joshmoore** [comment]
Green again.

**Comment by will-moore** [comment]
Tested with:
```
$ pip install awscli
$ mkdir 600124.zarr
$ aws s3 --endpoint-url https://minio-dev.openmicroscopy.org --no-sign-request --recursive cp s3://idr/zarr/v0.1/6001240.zarr 6001240.zarr
$ napari 6001240.zarr
```
👍

**Pull request merged**